SPHINXsys/src/shared/bodies/base_body.cpp

```diff
@@ -187,7 +187,7 @@ namespace SPH
 	{
 		TaggingParticleMethod tagging_particle_method = std::bind(&BodySurface::tagNearSurface, this, _1);
 		tagParticles(tagging_particle_method);
-		//std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
+		std::cout << "Number of surface particles : " << body_part_particles_.size() << std::endl;
```

Comment:
thanks for removing!

Reply:
I didn't change anything 😅 just merging upstream